src/main/steps/c100-rebuild/screening-questions/permission/content.ts

```diff
@@ -0,0 +1,83 @@
+import { YesOrNo } from '../../../../app/case/definition';
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { isFieldFilledIn } from '../../../../app/form/validation';
+
+const en = () => ({
+  title: 'Is there any reason that you would need permission from the court to make this application?',
+  paragraphsOne: [
+    'In some cases, the court will need to give permission before the application can be made.',
+    'You will not need permission if you are the:',
+  ],
+  bulletPoints: ['parent', 'guardian', 'special guardian'],
+  paragraphsTwo: [
+    'If you are not sure if you need permission, <a  target="_blank" href="https://www.gov.uk/government/publications/family-court-applications-that-involve-children-cb1">see Section C of the Form CB1 guidance on GOV.UK</a>',
+  ],
+  one: 'Yes',
+  two: 'No',
+  errors: {
+    sq_courtPermissionRequired: {
+      required:
+        'Select yes if there is any reason why you would need permission from the court to make this application',
+    },
+  },
+});
+
+const cy = () => ({
+  title: 'Is there any reason that you would need permission from the court to make this application? - welsh',
+  paragraphsOne: [
+    'In some cases, the court will need to give permission before the application can be made. - welsh',
+    'You will not need permission if you are the: - welsh',
+  ],
+  bulletPoints: ['parent - welsh', 'guardian - welsh', 'special guardian - welsh'],
+  paragraphsTwo: [
+    'If you are not sure if you need permission, <a  target="_blank" href="https://www.gov.uk/government/publications/family-court-applications-that-involve-children-cb1">see Section C of the Form CB1 guidance on GOV.UK</a> - welsh',
```

Comment:
@ShakiraKP , as per Jason T's comment on jira, the link is starting from Section C as below, where link stands for anchor tag:
see <link>Section C of the Form CB1 guidance</link>

Reply:
Done